src/pages/en/guides/upgrade-to/v2.mdx

```diff
@@ -0,0 +1,359 @@
+
+#### Replaced: `extendPlugins` -> `extendMarkdownConfig`
+
+In 1.0, `.mdx.extendPlugins` managed how your MDX files should inherit your Markdown plugin configuration. This setting was used to indicate whether all Markdown plugins (`markdown`), or only Astro's default plugins (`defaults`) should be applied to MDX.
+
+Astro 2.0 replaces `mdx.extendPlugins` with three new, independently-configurable options:
+
+- **[`mdx.extendMarkdownConfig`](/en/guides/integrations-guide/mdx/#extendmarkdownconfig)** (`true` by default) to inherit all of your Markdown configuration or none of it.
+- **`mdx.gfm`** (`true` by default) to enable or disable GitHub-Flavored Markdown in MDX.
+- **`mdx.smartypants`** (`true` by default) to enable or disable SmartyPants in MDX.
+
+ ##### What should I do?
+
+Delete `extendPlugins: 'markdown'` in your configuration. This is now default behavior. 
+
+```js del={6}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  mdx: {
+    extendPlugins: markdown,
+  }
+});
+```
+
+Replace `extendPlugins: 'defaults'` with `extendMarkdownConfig: false` and add the [separate options for GitHub-Flavored Markdown and SmartyPants](/en/guides/integrations-guide/mdx/#options-inherited-from-markdown-config) to enable these default plugins individually in MDX.
+
+```js del={6} ins={7-9}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  mdx: {
+    extendPlugins: defaults,
+    extendMarkdownConfig: false,
+    smartypants: true,
+    gfm: true,
+  }
+});
+```
+
+#### Added: More MDX config options to match Markdown
+
+Setting `extendMarkdownConfig: false` now allows you to individually set [every available Markdown configuration option](/en/reference/configuration-reference/#markdown-options) (except `drafts`) separately in your MDX integration configuration.
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import mdx from '@astrojs/mdx';
+
+export default defineConfig({
+  markdown: {
+    syntaxHighlight: 'prism',
+    remarkPlugins: [remarkPlugin1],
+    gfm: true,
+  },
+  integrations: [
+    mdx({
+      // `syntaxHighlight` inherited from Markdown
+
+      // Markdown `remarkPlugins` ignored,
+      // only `remarkPlugin2` applied.
+      remarkPlugins: [remarkPlugin2],
+      // `gfm` overridden to `false`
+      gfm: false,
+    })
+  ]
+});
+```
+
+##### What should I do?
+
+Revisit your Markdown and MDX configuration and compare your existing config with the new options available.
+
+:::tip[remember]
+- Astro's two default Markdown plugins are now individually configurable, and are not removed when you add other plugins. No more extending defaults!
+- The option to extend your Markdown plugin config to MDX has moved from a selection (just defaults or all plugins) to toggle (all or none).
+- Every Markdown config option (except drafts) can also be configured independently in MDX!
+:::
```

Comment:
Tried reworking this but… I can understand the first two too well and aren’t we just repeating what’s above? What do you think about removing this?

```suggestion
```